cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -609,13 +629,126 @@ method ExportImage(name: string, destination: string, compress: bool, tags: []st
+
+# ListPods returns a list of pods in no particular order.  There are
+# returned as an array of ListPodData structs.  See also [GetPod](#GetPod).
+method ListPod() -> (notimplemented: NotImplemented)
+
+# GetPod takes a name or ID of a pod and returns single ListPodData
+# structure.  A [PodNotFound](#PodNotFound) error will be returned if the pod cannot be found.
+# See also [ListPods](ListPods).
+method GetPod() -> (notimplemented: NotImplemented)
+
+# InspectPod takes the ID or name of a pod and returns one InspectPodData
+# structure.  A [PodNotFound](#PodNotFound) error will be returned if the pod cannot be found.
+# See also [ListPods](ListPods).
+# Note: this structure is not implemented because podman pod inspect has not been implemented.
+method InspectPod() -> (notimplemented: NotImplemented)
+
+# StartPod starts containers in a pod.  It takes the name or ID of pod.  It returns
+# the pod ID once started.  If the pod cannot be found, a [PodNotFound](#PodNotFound)
+# error will be returned.  If the pod has no containers, a [PodNoContainers](#PodNoContainers) 
+# error will be returned.  If the pod is successfully started, the pod ID will be returned.
+# See also [CreatePod](#CreatePod).
+# #### Example
+# ~~~
+# $ varlink call -m unix:/run/podman/io.projectatomic.podman/io.projectatomic.podman.StartPod '{"name": "135d71b9495f"}'
+# {
+#   "pod": "135d71b9495f7c3967f536edad57750bfdb569336cd107d8aabab45565ffcfb6"
+# }
+# ~~~
+method StartPod() -> (notimplemented: NotImplemented)
+
+# StopPod stops containers in a pod.  It takes the name or ID of a pod.
+# It returns the pod ID once stopped. If the pod cannot be found, a [PodNotFound](#PodNotFound)
+# error will be returned instead. See also [KillPod](KillPod).
+# #### Example
+# ~~~
+# $ varlink call -m unix:/run/podman/io.projectatomic.podman/io.projectatomic.podman.StopPod '{"name": "135d71b9495f"}'
+# {
+#   "pod": "135d71b9495f7c3967f536edad57750bfdb569336cd107d8aabab45565ffcfb6"
+# }
+# ~~~
+method StopPod() -> (notimplemented: NotImplemented)
+
+# RestartPod will restart containers in a pod given a pod name or ID. Containers in 
+# the pod that are running will be stopped, then all stopped containers will be run. 
+# If the pod cannot be found by name or ID, a [PodNotFound](#PodNotFound) error will be 
+# returned.  If the pod has no containers, a [PodNoContainers](#PodNoContainers) 
+# error will be returned.  If the pod is successfully started, the pod ID will be returned.
+# #### Example
+# ~~~
+# $ varlink call -m unix:/run/podman/io.projectatomic.podman/io.projectatomic.podman.RestartPod '{"name": "135d71b9495f"}'
+# {
+#   "pod": "135d71b9495f7c3967f536edad57750bfdb569336cd107d8aabab45565ffcfb6"
+# }
+# ~~~
+method RestartPod() -> (notimplemented: NotImplemented)
+
+# KillPod takes the name or ID of a pod as well as a signal to be applied to the pod.  Once the
+# containers in the pod have been killed, the pod's ID is returned.  If the pod cannot be found, a
```

Comment:
long term, it is better to not use the apostrophe possessive ... "the ID of the pod"